main.py

```diff
@@ -102,12 +118,14 @@ def publish_response(logger, req, resp):
         "workspace": req['params']['workspace'],
         "actions": req['params']['actions'],
         "response": resp,
+        "services": resp.get("services", None),
+        "updatetag": resp.get("updatetag", None),
     }
 
     pubsub_helper = PubSubLibrary()
 
     try:
-        if 'resultTopic' in req['params']:
+        if body.get('services', None):
```

Comment:
there should be 2 conditions here:
1. if services exist, push results to requestTopic
2. if resultTopic exists in params, push results to resultTopic